google-api-client/src/main/java/com/google/api/client/googleapis/services/EndToEndChecksumHandler.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.api.client.googleapis.services;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+
+/** This class provides End-to-End Checksum API for http protocol.
+ */
+public class EndToEndChecksumHandler {
+  /** The checksum http header on http requests */
+  public static final String HTTP_REQUEST_CHECKSUM_HEADER = "x-request-checksum-348659783";
+  /** The checksum http header on http responses */
+  public static final String HTTP_RESPONSE_CHECKSUM_HEADER = "x-response-checksum-348659783";
+
+  /**
+   * Create and return checksum as a string value for the input 'bytes'.
+   *
+   * @param bytes raw message for which the checksum is being computed
+   * @return      computed checksum as a <code>byte array</code>
```

Comment:
remove extra space; use {@code } instead of angle tags

Reply:
done